client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,12 +44,19 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Edit FEEL expressions in the properties panel</h4>
-          This release adds syntax highlighting and autocomplete for FEEL expressions to the properties panel.
+          <h4>Support for extension properties in Camunda Platform 8</h4>
+          We now support generic extension properties, which will be available for execution starting from Camunda Platform 8.1.
```

Comment:
Do users understand what extension properties are? May mention something that makes the link to C7 where users may know them already. But new users may not know what this is—maybe link the documentation

Reply:
Do we have documentation for this somewhere? Was not able to find something, yet: https://docs.camunda.io/docs/next/components/modeler/bpmn/tasks/

Reply:
What about simply

> You may already be familiar with this feature from Camunda Platform 7.